api-example.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Make sure to start the web UI with the following flags:
+#
+# python server.py --model MODEL --listen --no-stream
```

Comment:
Can we add a `--help` flag that will display this to the user?  Also, can we include the server ip address and port as arguments?  (or event "base url")?  I often run on a specific port, and I can easily see copying this script to a different server and using it to perform inference that way, so having a little extra config and a little extra help would be super useful.

Reply:
This has now been resolved by renaming the script and giving usage information and examples when it's run without arguments.